api/s2n.h

```diff
@@ -421,6 +421,97 @@ extern int s2n_connection_set_client_auth_type(struct s2n_connection *conn, s2n_
 S2N_API
 extern int s2n_connection_get_client_cert_chain(struct s2n_connection *conn, uint8_t **der_cert_chain_out, uint32_t *cert_chain_len);
 
+/**
+ * Sets the initial number of session tickets to send after a handshake.
```

Comment:
Let's be specific to avoid someone trying to use this with TLS1.2:
```suggestion
/**
 * Sets the initial number of TLS1.3 session tickets to send after a handshake.
```
Same for all the other new APIs that only affect TLS1.3.

Maybe also include a reference to `s2n_connection_add_new_tickets_to_send` as a related API that can be used later?

Reply:
I am trying to make these comments future-proof. If I add an explicit TLS1.3 statement it will be inaccurate in the future. I do agree though that adding TLS1.3 would make things more clear.

Reply:
Good point. But we still need to distinguish what doesn't effect earlier versions, so probably a note like "Only valid for <TLS1.3"?

Reply:
Added clarifying comment about >=TLS1.3 handshake